test/e2e/dynamic_provisioning_test.go

```diff
@@ -737,6 +737,104 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool) {
 		}
 		test.Run(cs, ns)
 	})
+
+	ginkgo.It("Should test pod failover with cordoning a node", func() {
+		skipIfUsingInTreeVolumePlugin()
+
+		t := dynamicProvisioningTestSuite{}
+
+		volume := testsuites.VolumeDetails{
+			ClaimSize: "10Gi",
+			VolumeMount: testsuites.VolumeMountDetails{
+				NameGenerate:      "test-volume-",
+				MountPathGenerate: "/mnt/test-",
+			},
+		}
+		pod := testsuites.PodDetails{
+			Cmd: convertToPowershellorCmdCommandIfNecessary("echo 'hello world' >> /mnt/test-1/data && while true; do sleep 3600; done"),
+			Volumes: t.normalizeVolumes([]testsuites.VolumeDetails{
+				{
+					ClaimSize: volume.ClaimSize,
+					MountOptions: []string{
+						"barrier=1",
+						"acl",
+					},
+					VolumeMount: volume.VolumeMount,
+				},
+			}, false),
+			IsWindows: isWindowsCluster,
+			UseCMD:    false,
+		}
+		podCheckCmd := []string{"cat", "/mnt/test-1/data"}
+		expectedString := "hello world\n"
+		if isWindowsCluster {
+			podCheckCmd = []string{"cmd", "/c", "type C:\\mnt\\test-1\\data.txt"}
+			expectedString = "hello world\r\n"
+		}
+
+		storageClassParameters := map[string]string{"skuName": "StandardSSD_LRS"}
```

Comment:
In the multi-az tests, we should use StandardSSD_ZRS otherwise the scheduler won't be able to start the pod on the other node because it is in a different zone that the disk. This is why the `pull-azuredisk-csi-driver-e2e-multi-az` fail.

```console
May 25 08:47:50.985: INFO: At 2021-05-25 08:47:40 +0000 UTC - event for azuredisk-volume-tester-cgncb-86c7fcdbff-smnp4: {default-scheduler } FailedScheduling: 0/3 nodes are available: 1 node(s) had taint {node-role.kubernetes.io/master: true}, that the pod didn't tolerate, 1 node(s) had taint {node.kubernetes.io/unschedulable: }, that the pod didn't tolerate, 1 node(s) had volume node affinity conflict.
```